From 730b80e691a4b9dd0e9727cfcd9806dfa542397b Mon Sep 17 00:00:00 2001
From: "commit-queue@webkit.org"
 <commit-queue@webkit.org@268f45cc-cd09-0410-ab3c-d52691b4dbfc>
Date: Fri, 4 Oct 2019 21:51:37 +0000
Subject: [PATCH] Fix build with icu 65.1
 https://bugs.webkit.org/show_bug.cgi?id=202600

Patch by Heiko Becker <heirecka@exherbo.org> on 2019-10-04
Reviewed by Konstantin Tokarev.

diff -urp qtwebkit-5.212.0-alpha3.orig/Source/WebCore/dom/Document.cpp qtwebkit-5.212.0-alpha3/Source/WebCore/dom/Document.cpp
--- qtwebkit-5.212.0-alpha3.orig/Source/WebCore/dom/Document.cpp	2019-06-26 11:25:02.000000000 -0500
+++ qtwebkit-5.212.0-alpha3/Source/WebCore/dom/Document.cpp	2019-11-13 07:09:37.655412611 -0600
@@ -4407,12 +4407,12 @@ static bool isValidNameNonASCII(const UC
     unsigned i = 0;
 
     UChar32 c;
-    U16_NEXT(characters, i, length, c)
+    U16_NEXT(characters, i, length, c);
     if (!isValidNameStart(c))
         return false;
 
     while (i < length) {
-        U16_NEXT(characters, i, length, c)
+        U16_NEXT(characters, i, length, c);
         if (!isValidNamePart(c))
             return false;
     }
@@ -4474,7 +4474,7 @@ bool Document::parseQualifiedName(const
 
     for (unsigned i = 0; i < length;) {
         UChar32 c;
-        U16_NEXT(qualifiedName, i, length, c)
+        U16_NEXT(qualifiedName, i, length, c);
         if (c == ':') {
             if (sawColon) {
                 ec = NAMESPACE_ERR;
